### ISSUE TITLE
Ghost is always in the way and I'm constantly accidentally interacting with it.

### ISSUE THREAD

**Issue by TheGuyWhoo** [00-Evan/shattered-pixel-dungeon#1209]
It is a ghost after all, you should be able to move through it. Wasting turns to get around it when it's in the way is pretty annoying.

**Comment by 00-Evan** [comment]
You can swap places with any ally by interacting with them.

**Comment by TheGuyWhoo** [comment]
Yes, but then I initiate the mini boss fight, which I don't always want at that particular moment.

**Comment by rpdelaney** [comment]
Talking to the ghost dismisses him and takes no time.

**Comment by TheGuyWhoo** [comment]
Yes, but as I mentioned this initiates the mini boss fight. It's really not ideal to immediately initiate the boss fight just to get sad ghost out of the way. Ideally you want to leave that for when the level is cleared out, in case you find any wands or armour or weapons that you can use in the fight and so you know where all the doors are and stuff. And also to prevent a huge pile up of enemies while you're trying to kite the boss or whatever... Lots of good reasons to hold of, and in that case you have to waste turns to try to get around sad ghost when he's in the way.

**Comment by rpdelaney** [comment]
Ah I misunderstood. Yeah that's annoying, especially on floor 3... I have also had frustrations as Huntress because the ghost absorbs my projectiles
